languages/elixir/exercises/concept/booleans/.docs/introduction.md

```diff
@@ -1,27 +1,27 @@
 Elixir represents true and false values with the boolean type. There are only two values -- _true_ and _false_. These values can be bound to a variable:
 
 ```elixir
-is_true = true
-is_false = false
+true_variable = true
+false_variable = false
 ```
 
 We can combine boolean values into expressions using boolean logical operators.  `and/2`, `or/2`, and `not/1` are commonly used in expressions to evaluate multiple boolean values to a single result.
 
 ```elixir
```

Comment:
You could consider shortening this is tiny bit to something like 3 example (and, or and not). We're not being exhaustive anyway and we can expect people to know about boolean logic.

Reply:
I want to highlight those operators so that people don't use `&&/2` `||/2` `!/1` because they behave differently, i can shorten it though

Reply:
We can always iterate on this later.